doc/nrf/protocols/matter/getting_started/custom_clusters.rst

```diff
@@ -0,0 +1,408 @@
+         * ``code`` - A unique identifier for the command within the cluster.
+         * ``name`` - The name of the command.
+         * ``optional`` - Indicates whether the command is optional.
+         * ``disableDefaultResponse`` - Indicates whether the default response to the command is disabled.
+         * ``response`` - The name of the response command, if any.
+         * ``description`` - A brief description of the command's purpose and functionality.
+         * ``arg`` - An argument for the command, specifying its name and type.
+
+      * ``<event>`` - An event definition within the cluster.
+
+         * ``source`` - Specifies whether the event originates from the client or server.
+         * ``code`` - A unique identifier for the event within the cluster.
+         * ``name`` - The name of the event.
+         * ``optional`` - Indicates whether the event is optional.
+         * ``description`` - A brief description of the event's purpose and functionality.
+         * ``arg`` - An argument for the event, specifying its name and type.
+
+      For example, the following XML code defines a cluster with one attribute, one command, and one event:
+
+      .. code-block:: xml
+
+         <?xml version="1.0"?>
+         <cluster>
+           <domain>General</domain>
+           <name>MyNewCluster</name>
+           <code>0xFFF1FC01</code>
+           <define>MY_NEW_CLUSTER</define>
+           <description>The MyNewCluster cluster showcases a cluster manufacturer extensions</description>
+           <attribute side="server" code="0x00" define="FLIP_FLOP" type="boolean" writable="true" default="false" optional="false">MyAttribute</attribute>
+           <command source="client" code="0x02" name="MyCommand" response="MyCommandResponse" optional="false">
+             <description>Command that takes two uint8 arguments and returns their sum.</description>
+             <arg name="arg1" type="int8u"/>
+             <arg name="arg2" type="int8u"/>
+           </command>
+           <event source="server" code="0x01" name="MyEvent" optional="false">
+             <description>Event that is generated by the server.</description>
+             <arg name="arg1" type="int8u"/>
+           </event>
+         </cluster>
+
+   .. tab:: ``<deviceType>``
+
+      ``<deviceType>`` elements define the device types that can be used in the cluster and consist of the following:
+
+      * ``<name>`` - The unique name of the device.
+      * ``<domain>`` - The domain to which the device belongs.
+      * ``<typeName>`` - The name of the device displayed in the zap tool.
+      * ``<profileId>`` - The profile ID of the device.
+
+         * ``editable`` - Indicates if the field can be modified.
+
+      * ``<deviceId>`` - The device ID.
+
+         * ``editable`` - Indicates if the field can be modified.
+
+      * ``<class>`` - The class of the device.
+         Can be ``Utility``, ``Simple``, or ``Node``.
+      * ``<scope>`` - The scope of the device.
```

Comment:
Class and scope are bolded as well.
![image](https://github.com/user-attachments/assets/4169daee-1b7b-4226-bb4c-4cb3e2162a9b)

Reply:
Same issue, one extra empty space before the next line after the bullet.